BitWatch/HomeView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  HomeView.swift
+//  BitWatch
+//
+//  Created by Sean Dunford on 1/11/15.
+//  Copyright (c) 2015 Razeware LLC. All rights reserved.
+//
+
+import UIKit
+import BitWatchKit
+
+public class HomeView: UIView {
+    var height: CGFloat!
+    var width: CGFloat!
+    var menuBlock: (() -> ())?
+    var beginBlock: (() -> ())?
+    
```

Comment:
So this is the equivalent of doing this in objective-c 
@property (nonatomic, copy) void (^menuBlock)();
@property (nonatomic, copy) void (^menuBlock)();

Basically we are making two block properties with no params and no return object. 

var menuBlock: ((params) -> (return))

Also, see these for more info:
http://thatthinginswift.com/completion-handlers/
http://fuckingblocksyntax.com/
http://fuckingswiftblocksyntax.com/